remotespark/kernels/kernelmagics.py

```diff
@@ -147,6 +147,16 @@ def info(self, line, cell="", local_ns=None):
 
         self.ipython_display.writeln("Session configs:\n\t{}\n".format(conf.get_session_properties(self.language)))
 
+        if self.session_started:
+            app_id = self.spark_controller.get_app_id()
+            driver_log_url = self.spark_controller.get_driver_log_url()
+            spark_ui_url = self.spark_controller.get_spark_ui_url()
+        else:
+            app_id = driver_log_url = spark_ui_url = "Session hasn't started."
+        self.ipython_display.writeln("Driver log:\n\t{}\n".format(driver_log_url))
```

Comment:
Driver logs shouldn't be gotten here, they should be printed as part of the `%%logs` command, or part of another command (`%%sparklogs`? `%%driverlogs`?)